website/cue/reference/components/sinks/http.cue

```diff
@@ -46,6 +46,7 @@ components: sinks: http: {
 				can_verify_certificate: true
 				can_verify_hostname:    true
 				enabled_default:        false
+				enabled_by_scheme:      true
```

Comment:
Is this correct? I've seen a few others that you modified where `enabled_default` is false but `enabled_by_scheme` is `true`.

Reply:
Ah, yes, good question. I left `enabled_default` as `true` if the sink has a default endpoint that uses HTTPS or is configured indirectly and defaults to HTTPS (like AWS region or Datadog site). In this case, the `http` sink has no default endpoint so I left `enabled_default` as `false`.

Reply:
Gotcha, that makes sense. 👍🏻